tests/unit/test_client.py

```diff
@@ -1056,3 +1060,51 @@ def test_request_headers_role_empty(mock_get_and_post):
     req.get("URL")
     _, get_kwargs = get.call_args
     assert_headers_with_roles(post_kwargs["headers"], None)
+
+
+def assert_headers_timezone(headers: Dict[str, str], timezone: Optional[str]):
```

Comment:
Make second parameter required instead of optional and move `get_localzone_name()` call to the test itself. Makes the test easier to follow.

Reply:
Sure.